.circleci/config.yml

```diff
@@ -196,6 +196,64 @@ jobs:
           working_directory: ./udata-front-plugins-helper
           command: npm run preversion
 
+  datagouv_components:
+    docker:
+      - image: cimg/node:lts
+    steps:
+      - checkout
+      - run:
+          name: Compute JS dependencies key
+          command: cat ./udata_front/theme/gouvfr/datagouv-components/package.json > datagouv-components-js.deps
+      - restore_cache:
+          keys:
+            - js-cache-{{ arch }}-{{ checksum "datagouv-components-js.deps" }}
+            - js-cache-{{ arch }}-{{ .Branch }}{{ .Environment.CIRCLE_TAG }}
+      - run:
+          name: Install NodeJS and dependencies
+          command: npm ci
+          working_directory: ./udata_front/theme/gouvfr/datagouv-components
+      - save_cache:
+          key: js-cache-{{ arch }}-{{ checksum "datagouv-components-js.deps" }}
+          paths:
+          - ./udata_front/theme/gouvfr/datagouv-components/node_modules
+      - save_cache:
+          key: js-cache-{{ arch }}-{{ .Branch }}{{ .Environment.CIRCLE_TAG }}
+          paths:
+          - ./udata_front/theme/gouvfr/datagouv-components/node_modules
+      - run:
+          name: Compile and test package
+          command: npm run preversion
+          working_directory: ./udata_front/theme/gouvfr/datagouv-components
+
+  publish_datagouv_components:
+    docker:
+      - image: cimg/node:lts
+    steps:
+      - checkout
+      - run:
+          name: Install NodeJS and dependencies
+          command: npm ci
+          working_directory: ./udata_front/theme/gouvfr/datagouv-components
+      - run:
+          name: Config npm registry
+          command: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> .npmrc
+          working_directory: ./udata_front/theme/gouvfr/datagouv-components
+      - run:
+          name: Publish on NPM
+          working_directory: ./udata_front/theme/gouvfr/datagouv-components
+          command: |
+            if [[ $CIRCLE_TAG ]]; then
+              # This is a tagged release
+              npm run publish-stable
+              npm run update-version -- prerelease --preid=dev
```

Comment:
The update-version in CI aren't pushed back on git right?

Reply:
No the current `dev` version isn't stored anywhere outside of NPM